recipes/nfft/meta.yaml

```diff
@@ -0,0 +1,49 @@
+#{% set version = "3.3.1" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: nfft
+  version: {{ version }}
+
+source:
+  fn: nfft-{{ version }}.tar.gz
+  # url: https://github.com/NFFT/nfft/releases/download/{{ version }}/nfft-{{ version }}.tar.gz
+  # md5: 8c0516e3252ff39a245e62f1a59d0165
+  url: https://www-user.tu-chemnitz.de/~potts/nfft/download/nfft-{{ version }}.tar.gz
+  md5: 8d9b1b708236297397ed1b70cace8b7b
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - fftw
+    - gcc  # [osx]
```

Comment:
Why only `OS X`? If you need OMP or Fortran the gcc package from the default channel is out current option for both Linux and OS X builds.